src/main/java/com/aws/greengrass/util/platforms/unix/linux/LinuxSystemResourceControllerV2.java

```diff
@@ -0,0 +1,132 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.aws.greengrass.util.platforms.unix.linux;
+
+import com.aws.greengrass.lifecyclemanager.GreengrassService;
+import com.aws.greengrass.logging.api.Logger;
+import com.aws.greengrass.logging.impl.LogManager;
+import com.aws.greengrass.util.Coerce;
+import org.apache.commons.lang3.StringUtils;
+
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.StandardOpenOption;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * https://www.kernel.org/doc/Documentation/cgroup-v2.txt
+ */
+public class LinuxSystemResourceControllerV2 extends LinuxSystemResourceController {
```

Comment:
Does this really need to be a different class? I was hoping that by properly abstracting the cgroup v1 and v2 using `CGroupSubSystemPath` interface, that we would not need the system resource controller to have 2 variants as well.

Reply:
We removed the “LinuxSystemResourceControllerV2.java” file according to last comments.